runtime/gc_base/GCExtensions.hpp

```diff
@@ -149,27 +149,45 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 	MM_ObjectAccessBarrier* accessBarrier;
 
 #if defined(J9VM_GC_FINALIZATION)
-	UDATA finalizeMainPriority; /**< cmd line option to set finalize main thread priority */
-	UDATA finalizeWorkerPriority; /**< cmd line option to set finalize worker thread priority */
+	uintptr_t finalizeMainPriority; /**< cmd line option to set finalize main thread priority */
+	uintptr_t finalizeWorkerPriority; /**< cmd line option to set finalize worker thread priority */
 #endif /* J9VM_GC_FINALIZATION */
 
 	MM_ClassLoaderManager* classLoaderManager; /**< Pointer to the gc's classloader manager to process classloaders/classes */
 #if defined(J9VM_GC_DYNAMIC_CLASS_UNLOADING)
-	UDATA deadClassLoaderCacheSize;
+	uintptr_t deadClassLoaderCacheSize;
```

Comment:
An annotation has been missed for this one.

Reply:
@amicic Please take a look at this

Reply:
threshold after which we flush class segments (not done for every class unloading, since it requires heap walk)